src/shared/lib/resources/affiliation.ts

```diff
@@ -24,7 +24,8 @@ export interface Affiliation {
 
 // Used when returning a list of the current user's affiliations
 export interface AffiliationSlim {
-  organizationName: string;
+  id: Id;
+  organization: Pick<Organization, 'id' | 'legalName'>;
```

Comment:
@sutherlanda FYI I expanded the `AffiliationSlim` type with necessary fields for the front-end.